GoogleForCreators/web-stories-wp
# POC: Editor: Multipage Canvas

## Context

This adds a preview of the pages before and after the current ones to either side of the canvas, if there is enough space. It adds as many pages, as there is room for.

## To-do

- [x] Which size should they pages be and with what margin?
- [x] Should the pages be active in any way? Can you click them? Can you drag stuff to them? Or from them?

## User-facing changes

| Window height | Screenshot |
|-|-|
| Narrow | <img width="2051" alt="Screen Shot 2022-06-10 at 17 32 42" src="https://user-images.githubusercontent.com/637548/173100760-757e80bf-b5fb-4db6-ab2d-b5706729b595.png"> |
| Medium | <img width="2051" alt="Screen Shot 2022-06-10 at 17 31 52" src="https://user-images.githubusercontent.com/637548/173100843-3114a49a-c01e-428c-b42d-f3d7f549dc57.png"> |
| Tall | <img width="2051" alt="Screen Shot 2022-06-10 at 17 32 15" src="https://user-images.githubusercontent.com/637548/173100871-e46e7f05-6c40-44a9-8c2e-1fea8b6397ab.png"> |

It even works in RTL:
<img width="2051" alt="Screen Shot 2022-06-10 at 17 40 17" src="https://user-images.githubusercontent.com/637548/173101738-88bd1cef-97ee-4409-a442-02dcd4ea478c.png">

## Testing Instructions

This PR can be tested by following these steps:

0. Enable _"Context pages"_ feature flag
1. Add multiple pages to a story (preferably with some content
2. Switch page a few times
3. Observe the previous and future pages in the margins of the current page
4. Try different browser sizes to see the changes

## Checklist




---

Partially addresses #9643
